hack/gen-flavors.sh

```diff
@@ -29,15 +29,15 @@ ci_dir="${REPO_ROOT}/templates/test/ci/"
 dev_dir="${REPO_ROOT}/templates/test/dev/"
 
 for name in $(find "${flavors_dir}"* -maxdepth 0 -type d -print0 | xargs -0 -I {} basename {} | grep -v base); do
-  ${KUSTOMIZE} build --load-restrictor LoadRestrictionsNone --reorder none "${flavors_dir}${name}" > "${REPO_ROOT}/templates/cluster-template-${name}.yaml"
+  ${KUSTOMIZE} build --load-restrictor LoadRestrictionsNone "${flavors_dir}${name}" > "${REPO_ROOT}/templates/cluster-template-${name}.yaml"
 done
 # move the default template to the default file expected by clusterctl
 mv "${REPO_ROOT}/templates/cluster-template-default.yaml" "${REPO_ROOT}/templates/cluster-template.yaml"
 
 for name in $(find "${ci_dir}"* -maxdepth 0 -type d -print0 | xargs -0 -I {} basename {} | grep -v patches); do
-  ${KUSTOMIZE} build --load-restrictor LoadRestrictionsNone --reorder none "${ci_dir}${name}" > "${ci_dir}cluster-template-${name}.yaml"
+  ${KUSTOMIZE} build --load-restrictor LoadRestrictionsNone "${ci_dir}${name}" > "${ci_dir}cluster-template-${name}.yaml"
 done
 
 for name in $(find "${dev_dir}"* -maxdepth 0 -type d -print0 | xargs -0 -I {} basename {} | grep -v patches); do
-  ${KUSTOMIZE} build --load-restrictor LoadRestrictionsNone --reorder none "${dev_dir}${name}" > "${dev_dir}cluster-template-${name}.yaml"
```

Comment:
Just for my own understanding: why is `--reorder none` removed here?

Reply:
`kustomize v5.4.1` complained on using `reorder` flag. It said `Flag --reorder has been deprecated, use the new 'sortOptions' field in kustomization.yaml instead`

Reply:
So we removed `--reorder` flag and are using `sortOptions` instead in the kustomization files